chroma-common/chroma-common.spec

```diff
@@ -0,0 +1,35 @@
+%{!?name: %define name chroma-common}
+%{?!version: %define version %(%{__python2} -c "from chroma_common import version; sys.stdout.write(version())")}
+%{?!package_release: %define package_release 1}
+%{?!python2_sitelib: %define python2_sitelib %(%{__python2} -c "from distutils.sysconfig import get_python_lib; import sys; sys.stdout.write(get_python_lib())")}
+
+Summary: Chroma Common
+Name: %{name}
+Version: %{version}
+Release: %{package_release}%{?dist}
+Source0: %{name}-%{version}.tar.gz
+License: Proprietary
```

Comment:
Licence is a requirement here and given we don't / haven't decided on what to change all the spec files to, will keep it consistent with the others and change them all in one patch